tools/clang/tools/dxcompiler/dxcshadersourceinfo.cpp

```diff
@@ -313,19 +314,18 @@ static std::vector<SourceFile> ComputeFileList(clang::CodeGenOptions &cgOpts,
       if (it->first->isValid() && !it->second->IsSystemFile) {
         // If main file, write that to metadata first.
         // Add the rest to filesMap to sort by name.
-        llvm::SmallString<128> NormalizedPath;
-        llvm::sys::path::native(it->first->getName(), NormalizedPath);
         if (cgOpts.MainFileName.compare(it->first->getName()) == 0) {
           SourceFile file;
-          file.Name = NormalizedPath.str();
+          file.Name = hlsl::NormalizePath(it->first->getName(),
```

Comment:
I can't figure out from the code why one of these cases need the `.` prefix, but the other one doesn't.

Reply:
I went ahead and got rid of the differences here. All paths regardless of main file or not is now normalized the same way. This makes things simpler in most cases.